### PR TITLE
Fixing some tiny bugs

### PR DESCRIPTION
**Pull request by phantom527**
Sending invite for code review.

### CODE REVIEW

File: 72NetworkBootstraped/Controllers/AccountController.cs [hunk 1]
```diff
@@ -126,7 +126,7 @@ public ActionResult SignUp(SignUpModel model)
           ModelState.AddModelError("", ErrorCodeToString(e.StatusCode));
         }
       }
-
+      ViewBag.Countries = new SelectList(Util.ListOfCountries());
```
Comment (phantom527) [inline 72NetworkBootstraped/Controllers/AccountController.cs:129]: Why is this required ?

Comment (mansoor-omrani) [inline 72NetworkBootstraped/Controllers/AccountController.cs:129]: When there is validation error in the form being posted back, ViewBag.Countries is empty and the View() encounteres error while being rendered.

File: 72NetworkBootstraped/Models/AccountModels.cs [hunk 2]
```diff
@@ -141,7 +141,8 @@ public class SignUpModel
 
     [Required]
     [Display(Name = "Mobile Phone")]
-    [RegularExpression(@"^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$", ErrorMessage = "Entered phone format is not valid.")]
```
Comment (phantom527) [inline 72NetworkBootstraped/Models/AccountModels.cs:144]: revert this back. It's not required
